src/Motor.Extensions.Hosting.Kafka/Options/KafkaConsumerOptions.cs

```diff
@@ -12,4 +12,5 @@ public KafkaConsumerOptions()
     public string? Topic { get; set; }
     public int CommitPeriod { get; set; } = 1000;
     public int MaxConcurrentMessages { get; set; } = 1000;
+    public bool SynchronousMessageHandling { get; init; }
```

Comment:
I think this option could use an XML comment to describe what it does exactly. I at least don't know what this does exactly.